hoomd_validation/init.py

```diff
@@ -13,8 +13,11 @@
 import lj_union
 import hard_disk
 import hard_sphere
+import simple_polygon
 
-subprojects = [alj_2d, lj_fluid, lj_union, hard_disk, hard_sphere]
+subprojects = [
+    alj_2d, lj_fluid, lj_union, hard_disk, hard_sphere, simple_polygon
```

Comment:
```suggestion
    alj_2d, lj_fluid, lj_union, hard_disk, hard_sphere, simple_polygon,
```
And run `pre-commit` to format one item per line.